src/components/PlayerCardContainer/CustomInput.js

```diff
@@ -3,9 +3,16 @@ import React, {useState} from "react";
 import './CustomInput.css'
 
 function CustomInput(props) {
+    if (props.hasGameStarted) {
         return (
-            <input type={props.type} className={props.classes} placeholder={props.placeHolder} onChange={props.changeHandler} />
-       )
+            <input type={props.type} className={props.classes} placeholder={props.placeHolder} onChange={props.changeHandler} readOnly/>
```

Comment:
If a user is never going to be able to type into this field, consider using a different element for display.  This allows you to have a different look and feel during the before-game-started state and the during-gameplay state.